common/djangoapps/philu_commands/management/commands/tests/test_fetch_course_structures.py

```diff
@@ -0,0 +1,29 @@
+"""
+Tests for 'fetch_course_structures' command
+"""
+import mock
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class TestFetchCourseStructure(ModuleStoreTestCase):
```

Comment:
Can we write one more test with invalid course id and assert that `generate_course_structure` is not called

`assert not generate_course_structure.called`